src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -2630,10 +2635,14 @@ export const handleDeleteTwilio: ControllerHandler<{ formId: string }> = (
         level: PermissionLevel.Delete,
       }),
     )
-    .andThen(({ msgSrvcName }) => {
+    .andThen((retrievedForm) => {
+      const { msgSrvcName } = retrievedForm
       if (!msgSrvcName) return okAsync(null)
 
-      return AdminFormService.deleteTwilioCredentials(formId, msgSrvcName)
+      return AdminFormService.deleteTwilioCredentials(
```

Comment:
since the msgSrvcName is a parameter of retrievedForm, do we really need two arguments in the `deleteTwilioCredentials` function? Should we just pass in retrievedForm and let the service handle it? (meaning service function can return null if form.msgSrvcName does not exist)

Reply:
You're right we can simplyfy this to just form now :D